app/Geocoding/GeocodingServiceProvider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Geocoding;
+
+use CultuurNet\UDB3\Container\AbstractServiceProvider;
+use CultuurNet\UDB3\Error\LoggerFactory;
+use CultuurNet\UDB3\Error\LoggerName;
+use Geocoder\Provider\GoogleMaps\GoogleMaps;
+use Geocoder\StatefulGeocoder;
+use Http\Adapter\Guzzle7\Client;
+
+final class GeocodingServiceProvider extends AbstractServiceProvider
+{
+    protected function getProvidedServiceNames(): array
+    {
+        return [
+            GeocodingService::class,
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $container->addShared(
+            GeocodingService::class,
+            function () use ($container) {
+                $googleMapsApiKey = isset($container->get('config')['google_maps_api_key']) ? $container->get('config')['google_maps_api_key'] : null;
```

Comment:
I think this can be changed to `$googleMapsApiKey = $container->get('config')['google_maps_api_key'] ?? null;`